src/topology/algebra/continuous_monoid_hom.lean

```diff
@@ -188,17 +188,19 @@ def coprod (f : continuous_monoid_hom A E) (g : continuous_monoid_hom B E) :
   inv := λ f, (inv E).comp f,
   mul_left_inv := λ f, ext (λ x, mul_left_inv (f x)) }
 
-instance : topological_space (continuous_monoid_hom A B) :=
+@[to_additive] instance : topological_space (continuous_monoid_hom A B) :=
 topological_space.induced to_continuous_map continuous_map.compact_open
 
 variables (A B C D E)
 
+@[to_additive]
 lemma is_inducing : inducing (to_continuous_map : continuous_monoid_hom A B → C(A, B)) := ⟨rfl⟩
```

Comment:
I know it is not in the scope of this PR, but I don't like this name because it doesn't follow the naming convention (we don't use `is_inducing` anywhere else). Could you please change it to `inducing_to_continuous_map`, or maybe just `inducing` but protected? Same for the two lemmas below